app/Http/Transformers/AssetsTransformer.php

```diff
@@ -160,4 +160,39 @@ public function transformAssignedTo($asset)
             'type' => $asset->assignedType()
         ] : null;
     }
+
+
+    public function transformRequestedAssets(Collection $assets, $total)
+    {
+        $array = array();
+        foreach ($assets as $asset) {
+            $array[] = self::transformRequestedAsset($asset);
+        }
+        return (new DatatablesTransformer)->transformDatatables($array, $total);
+    }
+
+    public function transformRequestedAsset(Asset $asset) {
+        $array = [
```

Comment:
As I mentioned above - is it possible to just re-use the existing asset listing and associated transforms to get basically the same output? Or can we add some new things - permissions, perhaps - to the existing listing and have it all still work out?